src/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Leaderboard-game</title>
+</head>
+<body>
+    <header>
+        <h1 class="header">Leaderboard</h1>
+    </header>
+    <main>
+        <!-- sections that shows score board -->
+        <section class="leaderboard">
+            <div class="header">
+                <h2 class="title">Recent scores</h2>
+                <button id="scores-refresh" type="button">Refresh</button>
+            </div>
+            <ul class="scores-list">
+                <!-- example list elements (scores) -->
+                <li>
+                    <p class="country">India</p>
+                    <p class="score-value">100</p>
+                </li>
+                <li>
+                    <p class="country">United States</p>
+                    <p class="score-value">22</p>
+                </li>
+                <li>
+                    <p class="country">Australia</p>
+                    <p class="score-value">59</p>
+                </li>
+                <li>
+                    <p class="country">England</p>
+                    <p class="score-value">77</p>
+                </li>
+                <li>
+                    <p class="country">Germany</p>
+                    <p class="score-value">121</p>
+                </li>
+                <li>
+                    <p class="country">Japan</p>
+                    <p class="score-value">109</p>
+                </li>
+            </ul>
```

Comment:
- [ Optional ] Good job so far, you've done well in this milestone. I think it would be better if you made sure that the `scores` list has the `zebra` stripped styling that is present on the project wireframe provided for this milestone. Please see the screenshot below for more details on this and rectify this.

## Your design
![Screenshot from 2023-08-03 00-07-33](https://github.com/the-faizmohammad/leaderboard-game/assets/16392046/dd1d9885-dba5-4d82-afb5-325df360fa01)

## Project wireframe
![leaderboard_wireframe](https://github.com/the-faizmohammad/leaderboard-game/assets/16392046/7a4d54b0-bf8e-4b16-92b2-920cb4d9efa9)

Reply:
Thank you for highlighting this, I have rectified as per instructions.